.gitignore

```diff
@@ -13,4 +13,5 @@ sassdoc/
 .sass-cache
 build-storybook.log
 chromatic-diagnostics.json
-storybook-static
\ No newline at end of file
+storybook-static
```

Comment:
@oluoluoxenfree to confirm if Storybook static is actually still used, and remove from gitignore if needed